cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,34 @@
+
+class IframePage {
+    private iFrameSite: string;
+    
+    constructor() {
+      this.iFrameSite = "https://www.w3schools.com/html/html_iframe.asp";
+    }
+  
+    public visit(): void {
+      cy.visit(this.iFrameSite);
+    }
+  
+    public getFrameTitle(htmlText: string): void {
+      cy.get('iframe').then($iframe => {
+        const $body = $iframe.contents().find('body') 
+        cy.wrap($body) 
+          .find('h1')
+          .should('have.text',htmlText)
+      })
```

Comment:
to avoid this code, the idea is to use the iframe command from the cypress-iframe dependency:
`cy.iframe(locatorForIFrame)`